examples/tutorials/hybrid_demucs_tutorial.py

```diff
@@ -0,0 +1,346 @@
+    out = fade(out)
+    final[:,:,:,prev:] += out
+    return final
+
+def plot_spectrogram(stft, title="Spectrogram"):
+    magnitude = stft.abs()
+    spectrogram = 20 * torch.log10(magnitude + 1e-8).numpy()
+    figure, axis = plt.subplots(1, 1)
+    img = axis.imshow(spectrogram, cmap="viridis", vmin=-100, vmax=0, origin="lower", aspect="auto")
+    figure.suptitle(title)
+    plt.colorbar(img, ax=axis)
+    plt.show()
+
+
+######################################################################
+# 5. Run Model
+# ------------
+# 
+# Finally, we run the model and store the separate source files in a
+# directory
+# 
+# As a test song, we will be using Strand Of Oaks by Spacestation from
+# MedleyDB (Creative Commons BY-NC-SA 4.0)
+# 
+# In order to test with a different song, the variable names and urls
+# below can be changed alongside with the parameters to test the song
+# separator in different ways.
+# 
+
+import requests
+
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+#We download the audio file from our storage. Feel free to download another file and use audio from a specific path
+SAMPLE_SONG = download_asset("tutorial-assets/hdemucs_mixture.wav")
+waveform, sample_rate = torchaudio.load(SAMPLE_SONG)
+store = waveform
+# waveform, sample_rate = torchaudio.load(SAMPLE_WAV) #replace SAMPLE_WAV with path
+
+#parameters
+segment: int = 10
+overlap = 0.1
+
+output_folder = "output"
+song_name = "test" #change accordingly
+mp3 = False #If track is MP3 file, change to True
+
+print(f"Separating track")
+
+ref = waveform.mean(0)
+waveform = (waveform - ref.mean()) / ref.std() #normalization
+
+sources = apply_model(model, waveform[None], device=device, 
+                      segment=segment, overlap=overlap,
+                      )[0]
+sources = sources * ref.std() + ref.mean()
+
+sources_list = model.sources
+sources = list(sources)
+
+audios = dict(zip(sources_list, sources))
+
+
+######################################################################
+# 5.1 Separate Track
+# ^^^^^^^^^^^^^^^^^^
+# 
+# The default set of pretrained weights that has been loaded has 4 sources
+# that it is separated into: drums, bass, other, and vocals in that order.
+# They have been stored into the dict “audios” and therefore can be
+# accessed there. For the four sources, there is a separate cell for each,
+# that will create the audio, the spectrogram graph, and also calculate
+# the sdr score.
+# 
+
+N_FFT = 4096
+N_HOP = 4
+stft = torchaudio.transforms.Spectrogram(
+    n_fft=N_FFT,
+    hop_length=N_HOP,
+    power=None,
+)
+
+
+######################################################################
+# 5.2 Original Track
+# ^^^^^^^^^^^^^^^^^^
+# 
+# This is the original track’s audio and spectrogram
+# 
+
+track = store
+segment = track[:, 10*sample_rate: 20*sample_rate]
+plot_spectrogram(stft(segment)[0], "Spectrogram Mixture")
```

Comment:
Spectrogram output is nice, but because the audio is 4 min and the time axis is shrunk, so IMO it does not convey as much interesting information, except vocal. Perhaps can we extract some portion where we can see the effectiveness of separation better?